src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/mgr-summary.pipe.spec.ts

```diff
@@ -31,7 +31,7 @@ describe('MgrSummaryPipe', () => {
     const expected = [
       { class: 'mgr-active-name', content: 'n/a active', titleText: '' },
       { class: 'card-text-line-break', content: '', titleText: '' },
-      { class: '', content: '0 standby', titleText: '' }
+      { class: 'mgr-standby-name', content: '0 standby', titleText: 'standby daemon: ' }
```

Comment:
Shouldn't the standby hover only appear if there are mgr's available?

```suggestion
      { class: 'mgr-standby-name', content: '0 standby', titleText: ' ' }
```

Reply:
Yeah , sure! This issue is still in progress, so I am currently making changes and will add this too.